x-pack/plugins/enterprise_search/server/routes/app_search/index.ts

```diff
@@ -46,4 +47,5 @@ export const registerAppSearchRoutes = (dependencies: RouteDependencies) => {
   registerOnboardingRoutes(dependencies);
   registerCrawlerRoutes(dependencies);
   registerCrawlerEntryPointRoutes(dependencies);
+  registerCrawlerCrawlRulesRoutes(dependencies);
```

Comment:
Maybe something to clean up after these are all merged in, but thinking about it I feel like maybe we should introduce a directory for these crawl routes

```
/app_search
-- /crawler
---- index.ts
---- crawler.ts
---- crawl_rules.ts
---- entry_points.ts

Maybe overkill, but I do think it could be good to develop some sort of pattern like this for features/views that require a lot of routes.

Reply:
I agree.

Reply:
Huge ++!